ui/components/app/nfts-tab/nfts-tab.js

```diff
@@ -60,7 +60,11 @@ export default function NftsTab({ onAddNFT }) {
         />
       ) : (
         <>
-          {isMainnet && !useNftDetection ? <NftsDetectionNotice /> : null}{' '}
+          {isMainnet && !useNftDetection ? (
+            <Box padding={4}>
+              <NftsDetectionNotice />
+            </Box>
+          ) : null}{' '}
```

Comment:
Is the `{' '}` really needed?

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/19806/commits/35fd2defb23542c07b9d56051004c569ee5200b5